packages/design-system-react/src/components/AvatarGroup/AvatarGroup.test.tsx

```diff
@@ -119,11 +120,12 @@ describe('AvatarGroup', () => {
       expect(() =>
         render(
           <AvatarGroup
-            variant={'Invalid' as any}
-            avatarPropsArr={[{ foo: 'bar' } as any]}
+            variant={'Invalid' as unknown as AvatarGroupVariant}
+            // @ts-expect-error - invalid variant
```

Comment:
We are expecting an error here so using `@ts-expect-error` is valid to resolve this violation